lib/NUPACK/emscripten/Utils.cpp

```diff
@@ -128,4 +128,153 @@ std::string getDotParens(bool pseudoknotted, const int seqlength, oneDnaStruct *
     {
        return singlestructure;
     }
+}
+
+std::string getDotParensNew(char* RNAseq, const int *thepairs, bool makeForEterna) {
```

Comment:
Seems like this is not actually used anywhere?

Reply:
I can take that out. I had that in there trying to see if there was a speed difference between what we are currently doing and the method I had pulled frm nupack. did not see a difference really maybe like a fraction of a second in our current methods favor so I did not put any more effort into it and forgot I had that there. I will remove next commit.

Reply:
i took this out and all references